test/tests/pbs_Rrecord_resources_used.py

```diff
@@ -316,123 +348,133 @@ def test_t5(self):
 
         j2 = Job(TEST_USER)
         j2.create_script(body=test)
-        j2.set_attributes({ATTR_r: 'n',ATTR_l + '.ncpus': 2})
+        j2.set_attributes({ATTR_r: 'n', ATTR_l + '.ncpus': 2})
         jid2 = self.server.submit(j2)
 
         j3 = Job(TEST_USER)
         j3.create_script(body=test)
-        j3.set_attributes({ATTR_r: 'y', ATTR_J: '1-4:1', ATTR_k: 'oe'})
+        j3.set_attributes({ATTR_J: '1-4', ATTR_k: 'oe'})
         jid3 = self.server.submit(j3)
 
         subjobs = self.server.status(JOB, id=jid3, extend='t')
         jid3s1 = subjobs[1]['id']
         """
         Verify that the jobs have started running.
         """
-        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid1, offset=5)
+        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid1)
         self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid2)
         self.server.expect(JOB, {ATTR_state: 'B'}, jid3)
         self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid3s1)
 
         """
-        Verify that the accounting logs have Resource_List.<resource> but no R records.
+        Verify that the accounting logs have Resource_List.<resource> but no
+        R records.
         """
-        m = self.server.accounting_match(msg='.*Resource_List.*', id=jid1, regexp=True)
+        m = self.server.accounting_match(
+            msg='.*Resource_List.*', id=jid1, regexp=True)
         self.assertNotEqual(m, None)
-        m = self.server.accounting_match(msg='.*R;' + jid1 + '.*resources_used.*', id=jid1, regexp=True)
+        m = self.server.accounting_match(
+            msg='.*R;' + jid1 + '.*resources_used.*', id=jid1, regexp=True)
         self.assertIsNone(m)
-        m = self.server.accounting_match(msg='.*Resource_List.*', id=jid2, regexp=True)
-        self.assertNotEqual(m,None)
-        m = self.server.accounting_match(msg='.*R;' + jid2 + '.*resources_used.*', id=jid2, regexp=True)
+        m = self.server.accounting_match(
+            msg='.*Resource_List.*', id=jid2, regexp=True)
+        self.assertNotEqual(m, None)
+        m = self.server.accounting_match(
+            msg='.*R;' + jid2 + '.*resources_used.*', id=jid2, regexp=True)
         self.assertIsNone(m)
-        m = self.server.accounting_match(msg='.*Resource_List.*', id=jid3s1, regexp=True)
+        m = self.server.accounting_match(
+            msg='.*Resource_List.*', id=jid3s1, regexp=True)
         self.assertNotEqual(m, None)
 
-        m = self.server.accounting_match(msg='.*R;' + re.escape(jid3s1) + '.*resources_used.*',
-        id=jid3s1, regexp=True)
+        m = self.server.accounting_match(msg='.*R;' + re.escape(jid3s1) +
+                                         '.*resources_used.*', id=jid3s1,
+                                         regexp=True)
         self.assertIsNone(m)
-        #sleeping for 30 seconds
+        """
+        sleep for 30 seconds so the jobs use some resources.
+        """
```

Comment:
I'm not sure having your jobs use some resources really buys you anything.  You're just checking for the existence of resources_used in the 'R' record.  If resources_used don't show right away, a much shorter sleep should provide you the same benefit.  I'm just trying to keep your tests zippy fast.